### ISSUE TITLE
Backend returns "null" hostname as TURL for WebDAV file transfer.

### ISSUE THREAD

**Issue by goiwai** [italiangrid/storm-puppet-module#14]
If we transfer a file with webdav through SRM like: 

```shell
gfal-copy -D "SRM PLUGIN:TURL_3RD_PARTY_PROTOCOLS=https" srm://srm.host.com:8444/srm/managerv2?SFN=/path/to/file /dev/null
```

the hostname in TURL is always resolved null, which looks like `https://null/path/to/file`, and eventually fails to transfer.

Because the file of `/etc/storm/backend-server/namespace.xml` contains multiple `<pool>` blocks for the filesystem configuration, i.e. not only gsiftp hosts but also webdav hosts are in the `<pool>`.

Removing lines between [L262 and L291 of the `namespace.xml.erb`](https://github.com/italiangrid/storm-puppet-module/blob/master/templates/etc/storm/backend-server/namespace.xml.erb#L262) will fix this issue. Perhaps this might create another issue due to the side effect.

I would be happy if you could consider this.

**Comment by enricovianello** [comment]
Hi Go,
first of all thanks for the notice.
I was able to reproduce it. It's completely my fault in the sense that I added this pool definition into StoRM Puppet module in order to be used within this not yet released v1.11.22 commit https://github.com/italiangrid/storm/commit/b7491d89490bfda90f7fe5dd6ac68ec35647434f that is included into StoRM 1.11.22 rpm available into StoRM beta repository.
I did some tests with StoRM v1.11.21 but it's clear that they weren't enough because, as you observed, introducing the lines you highlighted something broke at BE side.
My suggestion is:
- try to revert your StoRM puppet module to v3.3.1
- or trust our beta rpm of StoRM v1.11.22 (it's already used in some production environment here at INFN CNAF)
Let me know it this helps.

**Comment by goiwai** [comment]
Hello Enrico,

Thank you for your suggestion. I have confirmed that beta version 1.11.22 of StoRM creates the expected TURL with multiple `pool` blocks within the filesystem definition. For the moment, I would take the first suggestion at my site until the release of the v1.11.22. The issue: #14 now can be closed. Thank you very much.
